candidate-testing.js

```diff
@@ -1,35 +1,42 @@
-const input = require('readline-sync');
-
 // TODO 2: modify your quiz app to ask 5 questions //
 
 // TODO 1.1a: Define candidateName // 
-let candidateName;
+let candidateName = "";
 // TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
-let candidateAnswer;
+let question = "";
+let correctAnswer = "";
```

Comment:
I would go ahead and initialize with the correct answer that they provided. I believe it's Sally Ride. Then, you do not need line 27